packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -44,12 +44,11 @@ export const configSchema = {
     _description:
       'A list of identifier types to be displayed in the patient search results as banner tags. If no defaultIdentifierTypes are provided, the defaultIdentifier will be displayed.',
     // These values correspond to the Patient Clinic Number and National Unique Patient Identifier (NUPI) identifier types respectively
-    _default: ['b4d66522-11fc-45c7-83e3-39a1af21ae0d', 'f85081e2-b4be-4e48-b3a4-7994b69bb101'],
-  },
-  defaultIdentifier: {
-    _type: Type.String,
-    _description: 'Identifer type to be displayed when no defaultIdentifierTypes are provided. Default is OpenMRS ID.',
-    _default: 'OpenMRS ID',
+    _default: [
+      'b4d66522-11fc-45c7-83e3-39a1af21ae0d',
+      'f85081e2-b4be-4e48-b3a4-7994b69bb101',
+      '05a29f94-c0ed-11e2-94be-8c13b969e334',
+    ],
```

Comment:
IMO we shouldn't be hardcoding any identifier uuids here but this should come from the ref app configuration, but I know there's not agreement on that (and that this is existing code, not a change introduced as part of this code)

Reply:
What are the other identifiers then?

Reply:
So, these first two identifiers are Palladium-specific. Those are the Patient Clinic Number and the NUPI number. They should be provided via distro configuration instead of being hardcoded in the Patient Search config schema. @donaldkibet, can @usamaidrsk proceed to remove them from here?

Reply:
From the config file there is this comment, for the first two.
> These values correspond to the Patient Clinic Number and National Unique Patient Identifier (NUPI) identifier types respectively

Reply:
Thanks right @mogoodrich